styles/globals.css

```diff
@@ -45,40 +45,39 @@ html {
 }
 
 /* .herobtn { */
-  /* background-color: transparent !important; */
-  /* justify-content: center; */
-  /* width: 168px; */
-  /* height: 32px; */
-  /* padding: 8px 8px; */
-  /* margin-right: 10px; */
-  /* display: flex; */
-  /* border: 1px solid #E3E8EE;
+/* background-color: transparent !important; */
```

Comment:
There is no need to modify the css to just add to new tags to the style guide. I think your are adding here your work to modify the design of the table of contents but we should avoid it. Please use two different branches to fix each issue.